test/edu-benefits/pages/applicantService.unit.spec.jsx

```diff
@@ -4,10 +4,10 @@ import { expect } from 'chai';
 import sinon from 'sinon';
 import ReactTestUtils from 'react-addons-test-utils';
 
-import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
-import formConfig from '../../../../src/js/edu-benefits/5490/config/form';
+import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/edu-benefits/5490/config/form';
 
-describe('Edu 5490 applicantService', () => {
+describe('Edu 5490 (default) applicantService', () => {
```

Comment:
I think in here, since it's meant to test the code in edu-benefits/pages/applicantService.js, I'd import from that file and call the function to get the uiSchema. And use the schema from either 5490 or 5495.

It's the same result, but if we change something in applicantService.js, this test may continue to work and make us think we're covering it when we aren't.